functional/streams.py

```diff
@@ -47,6 +49,13 @@ def __call__(self, *args, no_wrap=None, **kwargs):
             args, engine, no_wrap=default_value(no_wrap, self.no_wrap, False)
         )
 
+    def of(self, *args, no_wrap=None, **kwargs):
+        """
+        Merge given the iterators firstly, then new the seq.
+        """
+        merged = list(chain.from_iterable(args))
```

Comment:
You should not use list to keep the iterator lazy

Reply:
yes, you're right. I'll check the args type.